### PR TITLE
fix: add missing items in item conversion map

### PR DESCRIPTION
**Pull request by ShadowShift**
Pumpkin Helmet uses the pumpkin texture (86) and was missed in the original run through

### CODE REVIEW

**Comment by ShadowShift** [comment]
Worth noting that in the api, this doesnt have a type. It is just displayed as "86".

File: Data-Storage/material_conversions.json [hunk 1]
```diff
@@ -522,7 +527,7 @@
   {
     "id": 257,
     "type": 0,
-    "name": "iron_shovel"
+    "name": "iron_pickaxe"
```
Comment (kristofbolyai) [inline Data-Storage/material_conversions.json:530]: really?

Comment (ShadowShift) [reply Data-Storage/material_conversions.json:530]: Yeahhhh... I'm gonna double check the rest when I get home. That one stood out cause it had forgotten pickaxe as the default spear :p

**Review by kristofbolyai — APPROVED** [review]
I've checked these changes and they are valid. If you find anything else, please PR it separately. Thanks!

**Pull request merged**